### PR TITLE
Update amp-in-pwa.md adding Polymer sample

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

File: content/docs/guides/pwa-amp/amp-in-pwa.md [hunk 1]
```diff
@@ -125,3 +125,8 @@ You can see the "AMP in PWA" pattern in action in the [React sample](https://git
 * Grab the source code here: [https://github.com/ampproject/amp-publisher-sample/tree/master/amp-pwa](https://github.com/ampproject/amp-publisher-sample/tree/master/amp-pwa)
 * Use the React component standalone via npm: [https://www.npmjs.com/package/react-amp-document](https://www.npmjs.com/package/react-amp-document)
 * See it in action here: [https://choumx.github.io/amp-pwa/](https://choumx.github.io/amp-pwa/) (best on your phone or mobile emulation)
+
+You can also see a sample of PWA and AMP using Polymer framework. It uses [amp-viewer](amp-viewer) to embed AMP pages.
```
Comment (bpaduch) [inline content/docs/guides/pwa-amp/amp-in-pwa.md]: Rewrite as: 
You can also see a sample of PWA and AMP that uses the Polymer framework. The sample uses an [amp-viewer](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-integration/integrating-viewer-with-amp-doc-guide.md) to embed AMP pages.

BTW, is that link accurate?  That is, does it use the the AMP Viewer API?  If it's not correct, please suggest an alternate link.

Comment (kul3r4) [inline content/docs/guides/pwa-amp/amp-in-pwa.md]: Hi Barb, it does, you can check in [https://github.com/Polymer/news/blob/amp/src/news-article.html](https://github.com/Polymer/news/blob/amp/src/news-article.html)

Comment (bpaduch) [inline content/docs/guides/pwa-amp/amp-in-pwa.md]: @kul3r4 - Sorry, I mean in the suggested rewrite that I propose, I link to https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-integration/integrating-viewer-with-amp-doc-guide.md, and I want to make sure that is the correct link.

In your writing you have `[amp-viewer](amp-viewer)`, which appears to go nowhere.

**Comment by kul3r4** [comment]
@bpaduch Sorry, I just updated the link.

**Pull request merged**